Fixed minor typo (#49053)
ray-project/ray: doc/source/tune/key-concepts.rst

Reviewer: Thanks for the fix!

 I noticed that there seems to be a discrepancy in the Function API and Class API examples below. For simplicity could we standardize on `a * (x ** 2) + b`. This would involve changing it in [this file](https://github.com/ray-project/ray/blob/2e4a126deecf84d2930ad772e763e206e2f5f7d2/doc/source/tune/doc_code/key_concepts.py#L8) as well.

Both are valid ways of defining a `trainable`, but the Function API is generally recommended and is used
throughout the rest of this guide.

> Let's say we want to optimize a simple objective function like ``a * (x ** 0.5) + b`` in which ``a`` and ``b`` are the

Reviewer 2: ```suggestion
Consider an example of optimizing a simple objective function like ``a * (x ** 0.5) + b`` in which ``a`` and ``b`` are the
```
